src/content/en/fundamentals/performance/optimizing-javascript/code-splitting/index.md

```diff
@@ -0,0 +1,745 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TBD
+
+{# wf_updated_on: 2018-07-30 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_blink_components: Blink>JavaScript #}
+
+# Reduce JavaScript Payloads with Code Splitting {: .page-title }
+
+{% include "web/_shared/contributors/jeremywagner.html" %}
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+**TL;DR:**
+
+- Modern apps often combine all of their JavaScript into a single, large bundle.
```

Comment:
Let's switch from apps to "sites" if that works for you. I've observed an increasing number of multi-page sites also leveraging code-splitting.